test/EnvironmentManagerTests/Helper.h

```diff
@@ -0,0 +1,167 @@
+﻿#pragma once
+#include "pch.h"
+
+using namespace winrt::Microsoft::ProjectReunion; // for EnvironmentManager
+using namespace winrt::Windows::Foundation::Collections; // for IMapView
+typedef IMapView<winrt::hstring, winrt::hstring> EnvironmentVariables;
+
+inline const wchar_t* USER_EV_REG_LOCATION = L"Environment";
+inline const wchar_t* MACHINE_EV_REG_LOCATION = L"SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Environment";
+
+inline EnvironmentVariables GetEnvironmentVariablesFromRegistry(HKEY hKey)
```

Comment:
Lots of comments but same as mentioned above. I'll wait for an update before commenting here

Reply:
Changed to a constexpr and renamed the helper file.